python/run-tests.py

```diff
@@ -59,9 +59,7 @@ def print_red(text):
 LOGGER = logging.getLogger()
 
 # Find out where the assembly jars are located.
-# Later, add back 2.12 to this list:
-# for scala in ["2.11", "2.12"]:
-for scala in ["2.11"]:
+for scala in ["2.11", "2.12"]:
     build_dir = os.path.join(SPARK_HOME, "assembly", "target", "scala-" + scala)
     if os.path.isdir(build_dir):
         SPARK_DIST_CLASSPATH = os.path.join(build_dir, "jars", "*")
```

Comment:
isn't this going to add both scala jar 2.11 and 2.12 into class path at the same time?

Reply:
@felixcheung . We assume that the Spark developers don't build both profiles without cleanup. This is a standard Spark way used since Spark 2.0.0.

Reply:
BTW, as we see here, although we officially deprecated Scala-2.11 at Spark 2.4.1, it seems that we don't test Scala-2.12 actually during Spark 2.4.2 vote process. The community is still on Scala-2.11.

Reply:
+1 on this. For next 2.4.3, we should have branch-2.4 default to Scala 2.12 as we will remove Scala 2.11 support in Spark 3.0

Reply:
Thanks, @dbtsai . +1 for that